### ISSUE TITLE
iOS Twitter app fails to load tweet threads/profiles after connecting to the outline server

### ISSUE THREAD

**Issue by rahmati** [Jigsaw-Code/outline-server#986]
The subject pretty much describes the issue. The home screen feed loads OK but I get a "Something went wrong error" when I click on a profile. Clicking on any tweet will open it correctly, but none of the replies/other tweets in the thread will load.

Things I have tried:
- Restarting the server doesn't help.
- The web version works fine.
- The issue happened on various models of iPhone
- Turning off outline will fix the issue.

**Comment by JonathanDCohen** [comment]
I'm curious what happens if you try to use Remote Desktop on your VM and go use twitter that way.  It's possible they may be blocking known Cloud IP addresses as part of bot or DDoS protections.

**Comment by rahmati** [comment]
I can try this and report back. However, in that case, shouldn't it also fail when I'm visiting Twitter through the mobile browser?
Also note that the initial tweet feed on homepage loads in the app without any issues.

**Comment by JonathanDCohen** [comment]
That's true, if it works in the mobile client through the same server then maybe this isn't the case

**Comment by rahmati** [comment]
No issues connecting to Twitter from the VM with or without the user-agent change.
